### PR TITLE
Update association example

### PR DESCRIPTION
**Pull request by davydovanton**
/cc @cllns

### CODE REVIEW

File: source/guides/models/associations.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ class AuthorRepository < Hanami::Repository
   end
 
   def find_with_books(id)
-    aggregate(:books).where(authors__id: id).as(Author).one
+    aggregate(:books).where(id: id).as(Author).one
```
Comment (AlfonsoUceda) [inline source/guides/models/associations.md]: @davydovanton are you sure of this? I mean, isn't there an error by ambiguity?

Comment (AlfonsoUceda) [inline source/guides/models/associations.md]: @davydovanton I tested in a new app, it works! forget what I said :)

**Comment by AlfonsoUceda** [comment]
LGTM 👍

File: source/guides/models/associations.md (hunk 1 above)
Comment (davydovanton) [inline source/guides/models/associations.md]: don't worry :) Thanks for merge!

**Pull request merged**